core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/WorkflowJSONExamples.scala

```diff
@@ -0,0 +1,59 @@
+package edu.uci.ics.amber.engine.e2e
+
+object WorkflowJSONExamples {
+
+  val headerlessCsvToSink =
+    """{"type":"ExecuteWorkflowRequest",
+      |"operators":[
+      |{"delimiter":",","header":true,"file path":"src/test/resources/CountrySalesData.csv","operatorID":"LocalCsvFileScan-operator","operatorType":"LocalCsvFileScan"},
+      |{"operatorID":"SimpleSink-operator","operatorType":"SimpleSink"}],
+      |"links":[
+      |{"origin":"LocalCsvFileScan-operator","destination":"SimpleSink-operator"}],
+      |"breakpoints":[]
+      |}""".stripMargin
+
+  val headerlessCsvToKeywordToSink =
```

Comment:
can we add a few test cases with aggregation? because aggregation requires shuffling, keyword search doesn't involve shuffling

Reply:
I will add a groupby